tchannel/tchannel.py

```diff
@@ -322,3 +322,9 @@ def advertise(self, routers=None, name=None, timeout=None,
         response = Response(json.loads(body), headers or {})
 
         raise gen.Return(response)
+
+    def stop(self, reason=None):
+        self._dep_tchannel.stop(reason)
```

Comment:
We need to stop building things into the old API. We can build *around* it, but we shouldn't also be adding features to the old API.